docs/dal/index.rst

```diff
@@ -146,10 +146,168 @@ Table Access Protocol
 
     -- `Table Access Protocol <https://www.ivoa.net/documents/TAP/>`_
 
+Getting started
+^^^^^^^^^^^^^^^
+
+Consider the following example for using TAP and ADQL, retrieving 5
+objects from the GAIA DR3 database, showing their id, position and
+mean G-band magnitude between 19 - 20:
+
 .. doctest-remote-data::
 
+    >>> import pyvo as vo
     >>> tap_service = vo.dal.TAPService("http://dc.g-vo.org/tap")
-    >>> tap_results = tap_service.search("SELECT TOP 10 * FROM ivoa.obscore")
+    >>> tap_service.search(
+    >>>        """ SELECT TOP 5 
+    >>>        source_id, ra, dec, phot_g_mean_mag 
+    >>>        FROM gaia.dr3lite
+    >>>        WHERE phot_g_mean_mag BETWEEN 19 AND 20
+    >>>        ORDER BY phot_g_mean_mag
+    >>>        """)
```

Comment:
Fix multi-line syntax, ensure that all doctest run and pass with locally running ``pytest docs/dal/ --remote-data``

```suggestion
    ...        """ SELECT TOP 5 
    ...        source_id, ra, dec, phot_g_mean_mag 
    ...        FROM gaia.dr3lite
    ...        WHERE phot_g_mean_mag BETWEEN 19 AND 20
    ...        ORDER BY phot_g_mean_mag
    ...        """)
```